product_packaging_container_deposit/models/container_deposit_order_line_mixin.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class OrderLineMixin(models.AbstractModel):
+    """This mixin should only be inherited by purchase.order.line and sale.order.line models."""
+
+    _name = "container.deposit.order.line.mixin"
+    _description = "Container Deposit Order Line Mixin"
+
+    is_container_deposit = fields.Boolean()
+
+    def _get_product_qty_field(self):
+        raise NotImplementedError()
+
+    def _get_product_qty_dlvd_rcvd_field(self):
+        raise NotImplementedError()
+
+    def _get_order_lines_container_deposit_quantities(self):
+        """
+        Returns a dict with quantity of container deposit
+            {
+                container_deposit_product: [quantity, qty_dlvd_rcvd]
+            }
+        """
```

Comment:
```suggestion
        """Get container deposit quantities by product.
        
        :return: a dict with quantity of container deposit
        
            {
                container_deposit_product: [quantity, $FIX_NAME]
            }
        """
```

https://peps.python.org/pep-0257/#multi-line-docstrings